src/main/kotlin/com/wolt/wm/training/bank/customer/models/Customer.kt

```diff
@@ -0,0 +1,27 @@
+package com.wolt.wm.training.bank.customer.models
+
+import java.util.*
+
+data class Customer(
+    val id: UUID,
+    val firstName: String,
+    val lastName: String,
+    val birthdate: Date,
```

Comment:
Prefer using date classes from java.time, rather than java util. For example, you can use https://docs.oracle.com/javase/8/docs/api/java/time/LocalDate.html